gubernator/view_build.py

```diff
@@ -127,17 +160,13 @@ def build_details(build_dir):
                   skipped: [name...],
                   passed: [name...]}
     """
-    started_fut = gcs_async.read(build_dir + '/started.json')
-    finished = gcs_async.read(build_dir + '/finished.json').get_result()
-    started = started_fut.get_result()
-    if finished and not started:
-        started = 'null'
-    if started and not finished:
-        finished = 'null'
-    elif not (started and finished):
-        return
-    started = json.loads(started)
-    finished = json.loads(finished)
+    started, finished = normalize_metadata(
+        gcs_async.read(build_dir + '/started.json'),
+        gcs_async.read(build_dir + '/finished.json')
+    )
+
+    if started is None and finished is None:
+        return started, finished, None
```

Comment:
weak preference: `return None` > `return None, None, None` > `return started, finished, None`.

Reply:
Eh, different return types in one method make my head hurt and complicates the API for callers so I think I'll keep this as-is.